clients/fides-js/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  // TODO: I really am just guessing at this. Find a better example to copypasta from
+  // Adapted from https://turbo.build/repo/docs/handbook/sharing-code/internal-packages
```

Comment:
Also adapted from the tsconfig.json in `fides-consent`, but that one is fairly minimal - all the real magic is in the rollup config. I nuked that ;)